### PR TITLE
feat: select component page

### PR DESCRIPTION
**Pull request by hlavkat**

### CODE REVIEW

File: src/routes/components/select/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	</div>
+	<div class="example-row">
+		<Typography variant="small" bold>1. compact size select</Typography>
+		<Select dimension="compact" label="Select label" />
+	</div>
+	<div class="example-row">
+		<Typography variant="small" bold>1. small size select</Typography>
+		<Select dimension="small" label="Select label" />
+	</div>
+{/snippet}
```
Comment (vojtechsimetka) [inline src/routes/components/select/+page.svelte:71]: Please add two options for each select and a placeholder.

File: src/routes/components/select/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+		<TabContent value="Preview">
+			<div class="preview-tabs preview-tab">
+				<Select {dimension} {label}>
+					<Option value="1">Option 1</Option>
+					<Option value="2">Option 2</Option>
+					<Option value="3">Option 3</Option>
+					<Option value="4">Option 4</Option>
+				</Select>
+			</div>
+		</TabContent>
```
Comment (vojtechsimetka) [inline src/routes/components/select/+page.svelte:98]: Add placeholder and helpertext

File: src/routes/components/select/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+{#snippet controls()}
+	<Select bind:value={dimension} label="Size" helperText={helperTextSelectSizes}>
+		<Option value="default">Default</Option>
+		<Option value="large">Large</Option>
+		<Option value="compact">Compact</Option>
+		<Option value="small">Small</Option>
+	</Select>
+	<Input bind:value={label} label="Add label" />
+{/snippet}
```
Comment (vojtechsimetka) [inline src/routes/components/select/+page.svelte:85]: Add configuration for helper text and placeholder

**Review by vojtechsimetka — CHANGES_REQUESTED** [review]

File: src/routes/components/select/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+	</TabBar>
+{/snippet}
+
+{#snippet sizes()}
+	<Typography></Typography>
+{/snippet}
+
+<ComponentTemplate
+	name="Select"
+	tagline="Lorem ipsum, dolor sit amet consectetur adipisicing elit. Molestiae, eveniet."
```
Comment (vojtechsimetka) [inline src/routes/components/select/+page.svelte:129]:
```suggestion
	tagline="Select components allow people to choose one option from a dropdown list"
```

**Review by agazso — APPROVED** [review]

**Pull request merged**